tests/similarItems.test.ts

```diff
@@ -100,9 +104,33 @@ describe('Gear', () => {
 		body: 'Elite void top'
 	});
 	test('', () => {
+		expect(testGear8.hasEquipped('Scythe of vitur', true)).toBeTruthy();
+		expect(testGear8.hasEquipped('Void knight top', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Kodai wand', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Black mask (i)', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Scythe of vitur', true)).toBeTruthy();
 		expect(testGear8.hasEquipped('Void knight top', true)).toBeTruthy();
 	});
+
+	for (const [baseItem, slot, similarItems] of [
+		['Scythe of vitur', '2h', ['Holy scythe of vitur', 'Sanguine scythe of vitur']],
+		[
+			'Scythe of vitur (uncharged)',
+			'2h',
+			['Holy scythe of vitur (uncharged)', 'Sanguine scythe of vitur (uncharged)']
+		],
+		['Sanguinesti staff', 'weapon', ['Holy sanguinesti staff']],
+		['Sanguinesti staff (uncharged)', 'weapon', ['Holy sanguinesti staff (uncharged)']]
+	] as const) {
+		for (const simItem of resolveItems(similarItems as unknown as string[])) {
+			const gear = new Gear({
+				[slot]: itemNameFromID(simItem)
+			});
+			const is = gear.hasEquipped(itemID(baseItem));
+			if (!is) {
+				throw new Error(`${baseItem} didn't match ${itemNameFromID(simItem)}`);
+			}
+			expect(is).toEqual(true);
+		}
+	}
```

Comment:
I'd suggest one more (opposite) check to make sure hasEquipped('Charged variant') does NOT match an equipped('Uncharged variant')

```ts
const gear = new Gear({ '2h': 'Scythe of vitur (uncharged)' });
expect(gear.hasEquipped('Scythe of vitur')).toEqual(false);
```